website/docs/r/s3_bucket_replication_configuration.html.markdown

```diff
@@ -216,6 +216,8 @@ The following arguments are supported:
 
 * `bucket` - (Required) The name of the source S3 bucket you want Amazon S3 to monitor.
 * `role` - (Required) The ARN of the IAM role for Amazon S3 to assume when replicating the objects.
+* `token` - (Optional) A token to allow replication to be enabled on an Object Lock-enabled bucket. You must contact AWS support for the bucket's "Object Lock token".
```

Comment:
Very minor nit: do you mind moving `token` just below `rule` to maintain the order?

Reply:
Going to edit this as we're going to do a release today and would love to get this as it's ready to go 🚀